system/jlib/jtrace.cpp

```diff
@@ -916,8 +976,9 @@ class CSpan : public CInterfaceOf<ISpan>
         //The localid is passed as the callerid for the client request....
         clientHeaders->setNonEmptyProp(kCallerIdHttpHeaderName, queryLocalId());
 
-        if (span == nullptr)
-            return;
+        //for conditional spans, we can't rely on having an actual span object
```

Comment:
If conditional spans wish to report attributes, we can't fall out based on validity of underlying span obj